CMakeLists.txt

```diff
@@ -286,7 +286,7 @@ if (BUILD_TESTING)
         )
     endforeach()
 
-    file(GLOB_RECURSE test_py_files "${CMAKE_SOURCE_DIR}/Utest/*runtest_*.py")
+    file(GLOB_RECURSE test_py_files "${CMAKE_SOURCE_DIR}/systemtest/*runtest_*.py")
```

Comment:
Is there a file which matches `*runtest_*.py`?

Reply:
Yes ;) for example: `/systemtest/juelich_tests/test_1/runtest_juelich_1.py`